libcore/ConnectServerDialog.vala

```diff
@@ -19,7 +19,11 @@
 * Authored by: Corentin Noël <corentin@elementary.io>
 */
 
-public class PF.ConnectServerDialog : Granite.Dialog {
+public class PF.ConnectServerDialog : Gtk.Widget {
+// Gtk.Dialog is final type and cannot be subclassed.
+// public class PF.ConnectServerDialog : Granite.Dialog {
```

Comment:
Since we don't use the dialog action area here anyways, maybe it would be better to just make this a `Gtk.Window`subclass and `add_css_class ("dialog")`. I don't think we're actually using any `Granite.Dialog` features here, we just want the dialog styling

Reply:
Done